src/components/UserAccess/RBListRow.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { useRoleMap } from '../../hooks/useRole';
+import { RowFunctionArgs, TableData } from '../../shared';
+import ActionMenu from '../../shared/components/action-menu/ActionMenu';
+import { RoleBinding } from '../../types';
+import { rbTableColumnClasses } from './RBListHeader';
+import { useRBActions } from './user-access-actions';
+
+export const RBListRow: React.FC<React.PropsWithChildren<RowFunctionArgs<RoleBinding>>> = ({
+  obj,
+}) => {
+  const actions = useRBActions(obj);
+  const [roleMap, roleMapLoading] = useRoleMap();
+
+  if (roleMapLoading) {
+    return (
+      <Bullseye>
+        <Spinner data-test="spinner" />
+      </Bullseye>
+    );
+  }
```

Comment:
We should use a skeleton instead of a spinner and only in the column that needs loading state